genie-ddl/src/main/sql/postgresql/schema.postgresql.sql

```diff
@@ -209,8 +221,8 @@ CREATE TABLE job_requests (
     command_args character varying(10000) NOT NULL,
     group_name character varying(255) DEFAULT NULL::character varying,
     setup_file character varying(1024) DEFAULT NULL::character varying,
-    cluster_criterias character varying(2048) DEFAULT '[]'::character varying NOT NULL,
-    command_criteria character varying(1024) DEFAULT '[]'::character varying NOT NULL,
+    cluster_criterias text DEFAULT ''::character varying NOT NULL,
```

Comment:
It seems here that Postgres allows you to have defaults for their text field so why not just keep it at `[]`?